src/utils/logger.js

```diff
@@ -42,7 +42,10 @@ export const logger = Logger({
                 {}
             );
 
-        return request(method, url, {
+        const urlNew =
+            getDisableSetCookie() === true ? `${url}?disable-set-cookie=true&features=disable-set-cookie` : url;
```

Comment:
We can omit the `=== true` as that is already implied. Also consider giving this a more meaningful name to improve clarity such as `urlWithCookieParams`
```suggestion
        const urlWithCookieParams =
            getDisableSetCookie() ? `${url}?disable-set-cookie=true&features=disable-set-cookie` : url;
```